.github/ISSUE_TEMPLATE/0-bug-report.yaml

```diff
@@ -0,0 +1,311 @@
+name: New Bug Report
+description: File a bug report
+title: "Product-Area-Here: short description of bug here"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please try to give your issue a good title. Try using the product-area where you are having an issue and a brief description of the problem. Like this: 
+        - `Dashboards: Template Variables break when I do X` or 
+        - `Alerting: message templating plus Slack channel breaks when I do X`
+  - type: markdown
+    attributes:
+      value: |
+        **HINT:** Have you tried [searching](https://github.com/grafana/grafana/issues) for similar issues? Duplicate issues are common.
+
+        **Are you reporting a security vulnerability?** [Submit it here instead](https://github.com/grafana/grafana/security/policy).
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: textarea
+    id: bug-describe
+    attributes:
+      label: |
+        
+        # 1 / 3: What went wrong?
+        
+      description: |
+        #
+        Describe your bug. What happened? What did you expect to happen?
+
+        **Hot Tip:** Record your screen and add it here as a gif. 
+      placeholder: Tell us what you see!
+      value: |
+        **What happened**:
+
+          - 
+
+        **What did you expect to happen**:
+
+          - 
+
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: textarea
+    id: bug-repro
+    attributes:
+      label: |
+        
+        # 2 / 3: How do we reproduce it?
+        
+      description: |
+        #
+
+        Whenever possible, please provide **detailed** steps for reproducing your bug.
+
+        **This is very helpful info** 
+      placeholder: "Step 1:..."
+      value: |
+        **Step 1**:
+        
+        - Open Grafana and do X 
+
+        **Step 2**:
+
+        - Now click button Y
+
+        **Step 3**:
+
+        - Wait for the browser to crash. Error message says: "Error..."
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: dropdown
+    id: gf-version
+    attributes:
+      label: |
+        
+        # 3 / 3: What Grafana version are you using?
+        
+      description: |
+        #
+        - You can find your Grafana version:
+          - inside the UI [here](https://share.cleanshot.com/wVTvrKhy), [here](https://share.cleanshot.com/5qg3RKF1), or [here](https://share.cleanshot.com/988MQZz1), depending on your platform and access level.
```

Comment:
I’d suggest placing all of this in the docs instead and having a single link to “how to figure out your version”. Linking to screenshots makes for poor accessibility and confusing interface.